quora-service/src/main/java/com/upgrad/quora/service/business/QuestionBusinessService.java

```diff
@@ -0,0 +1,150 @@
+package com.upgrad.quora.service.business;
+
+import com.upgrad.quora.service.dao.QuestionDao;
+import com.upgrad.quora.service.dao.UserAuthTokenDao;
+import com.upgrad.quora.service.dao.UserDao;
+import com.upgrad.quora.service.entity.*;
+import com.upgrad.quora.service.exception.AuthorizationFailedException;
+import com.upgrad.quora.service.exception.InvalidQuestionException;
+import com.upgrad.quora.service.exception.UserNotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+/**
+ * Author : Harika Etamukkala
+ * QuestionBusinessService will contain business logic related to Question feature
+ */
+@Service
+public class QuestionBusinessService {
+
+    public static final String USER_HAS_NOT_SIGNED_IN = "User has not signed in";
+    public static final String USER_IS_SIGNED_OUT_SIGN_IN_FIRST_TO_POST_A_QUESTION = "User is signed out.Sign in first to post a question";
```

Comment:
This violates encapsulation principal of Object oriented programming. Make all public static variables private